microsoft/testsuites/kselftest/kselftest.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import PurePath, PurePosixPath
+from dataclasses import dataclass
+from lisa.executable import Tool
+from lisa.messages import TestStatus
+from typing import Any
+from lisa.tools.remote_copy import RemoteCopy
+from lisa.tools.tar import Tar
+import os
+from lisa.util import MissingPackagesException
+
+@dataclass
+class KselftestResult:
+    version: str = ""
+    name: str = ""
+    status: TestStatus = TestStatus.QUEUED
+    exit_value: int = 0
+
+class Kselftest(Tool):
+    @property
+    def command(self) -> str:
+        return str(self._command)
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
+        # command to run kselftests
+        self._command = self.get_tool_path(use_global=True) / "kselftest-packages/run_kselftest.sh > kselftest-results.txt"
+        # tar file path specified in yml
+        self._tar_file_path = kwargs.pop("kselftest_file_path", "")
```

Comment:
`kselftest_file_path` should be an argument in function rather than from kwarg from clarity

Reply:
> `kselftest_file_path` should be an argument in function rather than from kwarg from clarity

kselftest_file_path is now a function arg instead of kwarg, thanks for helping with this :)